Teams/04 Fundamentals of Microsoft Teams/Demos/01-teams-app-yeoman/README.md

```diff
@@ -80,67 +84,55 @@ This exercise introduces the Yeoman generator and its capabilities for scaffoldi
 
 1. Type `yo teams` and press **Enter**.
 
-    ![Screenshot of Yeoman Teams generator.](../../Images/Exercise1-01.png)
-
 1. When prompted, accept the default **teams-app-1** as your solution name and press **Enter**.
 
 1. Select **Use the current folder** for the file location and select **Enter**. The next set of prompts asks for specific information about your Microsoft Teams app:
     - Accept the default **teams app1** as the name of your Microsoft Teams app project and press **Enter**.
     - Enter your name and press **Enter**.
+    - Accept the default selection of **v1.4** for the manifest version you would like to use and press **Enter**.
```

Comment:
Should just be "Accept the default selection for the manifest version" we're on 1.5 already, and will be on 1.6 before the next refresh

Reply:
The "default" comment is in regards to the [Yeoman generator](https://github.com/officedev/generator-teams). The generator provides the following choices for the manifest version:

![image](https://user-images.githubusercontent.com/8008013/59384595-2629e980-8d28-11e9-95aa-c877be402aed.png)

Reply:
Right, but the Yeoman generator will support version 1.5 in a very short amount of time, and probably 1.6 before the next refresh of the content. No reason to say more than "accept the default selection for the manifest version"

Reply:
Ah, right. I thought you wanted language around 1.5 in the instructions. Updated to 

"Accept the default selection for the manifest version..."